packages/teuchos/comm/src/Teuchos_StackedTimer.cpp

```diff
@@ -255,6 +283,20 @@ StackedTimer::computeColumnWidthsForAligment(std::string prefix,
           os << "}";
         alignments_.max_ = std::max(alignments_.max_,os.str().size());
       }
+      if (procmin_.size()) {
+        std::ostringstream os;
+        os << ", proc min=" << procmin_[i];
+        if (active_[i] <= 1)
+          os << "}";
+        alignments_.procmin_ = std::min(alignments_.procmin_,os.str().size());
+      }
+      if (procmax_.size()) {
+        std::ostringstream os;
+        os << ", proc max=" << procmax_[i];
+        if (active_[i] <= 1)
+          os << "}";
+        alignments_.procmax_ = std::max(alignments_.procmax_,os.str().size());
+      }
       if (active_[i]>1) {
         std::ostringstream os;
         os << ", std dev=" << sqrt(std::max<double>(sum_sq_[i]-sum_[i]*sum_[i]/active_[i],0.0)/(active_[i]-1));
```

Comment:
In some cases, the standard library forbids specifying the template arguments of a standard algorithm.  I think it's allowed for `max` etc., but it's poor form in my opinion.  (General best practice is to rely on template argument deduction for nonmember functions.)  It would be better to cast the first argument to `double`.  (`0.0` already has type `double`.)  Doing so would help avoid any truncation if `sum_[i]` and `active_[i]` both have integer types.